scripts/azureml-assets/azureml/assets/validate_assets.py

```diff
@@ -50,6 +50,9 @@
     f"(?:-(?:{GPU_DRIVER_VERSION}|gpu))?",
 ]))
 
+# Dockerfile convention
+DOCKERFILE_IMAGE_PATTERN = re.compile(r"^FROM\s+mcr\.microsoft\.com\/azureml\/curated\/")

```

Comment:
No need to escape `/`s, and let's be more generic:
```suggestion
DOCKERFILE_IMAGE_PATTERN = re.compile(r"^FROM\s+mcr\.microsoft\.com/azureml/curated/", re.IGNORECASE | re.MULTILINE)
```